docs/source/guides/writer/chapters/recipes.rst

```diff
@@ -57,3 +57,31 @@ be executed::
 .. tip:: As a possible integration strategy with existing tests, you
          can have one or more runnable recipe files that are passed
          to Avocado to be executed.
+
+Combining multiple recipes in a single file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Avocado also ships with a slightly difference resolver, called
+``runnables-recipe``.  It reads a recipe file that, instead of
+containing one single runnable, contains (potentially) many.
+It should contain nothing more than an a list of runnables, as per the
+following schema:
+
+.. literalinclude:: ../../../../../avocado/schemas/runnables-recipe.schema.json
+
+For instance, to run both ``/bin/true`` and ``/bin/false``, you can
+define a file like::
+
+  [
```

Comment:
I would use `literalinclude::` from examples, instead of hard coded example. It will be easier for future maintenance.